core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -24,9 +27,11 @@
     private final EncodingManager encodingManager;
     private final int motorVehicleInt;
 
+    private final double DELTA = 0.1;
+
     public DataFlagEncoderTest() {
         encoder = new DataFlagEncoder();
-        encodingManager = new EncodingManager(encoder);
+        encodingManager = new EncodingManager(Arrays.asList(encoder), 8);
```

Comment:
Would be great if we could make the number of encoders configureable. Maybe if we pass a heightlimit of 0 via the Configmap to the DataflagEncoder, we don't register the heigthEncoder. WDYT?